web-scrapers/qualifications.json

```diff
@@ -0,0 +1,19 @@
+{"qualifications":[
```

Comment:
qualifications.json doesn't seem to be used anywhere. Is this meant for something? If not it could be good to include in the dataset description if this is being extracted from somewhere?

Reply:
it was used in a previous commit, when we still had each file as a dataset, to create the descriptions for the datasets. The information in this file is not scraped, but only copied from the SQA website.

However, now the datasets are different, so I don't know how to use it anymore. I would delete the file from the PR, unless we find a good use for it.

Reply:
agree with delete.